polyglot.js

```diff
@@ -221,6 +230,25 @@ class PolyGlot {
+                break;
+            case "pf1":
+            case "pf2e":
+                this.alphabets = {common:'130% Thorass',abyssal:'150% Barazhad',auran:'100% OldeThorass',azlanti:'120% Tengwar',boggard:'120% Semphari',celestial:'180% Celestial',outwordly:'200% ArCiela',draconic:'170% Iokharic',dwarvish:'120% Dethek',drowic:'150% HighDrowic',druidic:'100% AngloSax',dziriak:'250% Pulsian',giant:'120% MeroiticDemotic',gnoll:'150% Kargi',elvish:'150% Espruar',erutaki:'120% Tuzluca',garundi:'120% Qijomi',infernal:'230% Infernal',jistka:'120% Valmaric',jungle:'120% JungleSlang',kelish:'170% HighschoolRunes',oriental:'120% Oriental',requian:'150% Reanaarian',serpent:'120% Ophidian',signs:'170% FingerAlphabet',sylvan:'150% OldeEspruar',thassilonian:'150% Thassilonian',utopian:'140% MarasEye'}
+                this.tongues = {_default:'common',aboleth:'outwordly',abyssal:'abyssal',aklo:'serpent',algollthu:'outwordly',anadi:'jungle',aquan:'auran',arboreal:'sylvan',auran:'auran',azlanti:"azlanti",boggard:"boggard",caligni:"drowic",celestial:"celestial",cyclops:"giant",daemonic:"infernal",dark:"drowic",destrachan:"outwordly",draconic:"draconic",drowsign:"signs",druidic:"druidic",dwarven:"dwarvish",dziriak:"dziriak",elven:"elvish",erutaki:"erutaki",garundi:"garundi",giant:"giant",gnoll:"gnoll",gnome:"dwarvish",gnomish:"dwarvish",goblin:"gnoll",grippli:"boggard",hallit:"azlanti",ignan:"dwarvish",iruxi: "boggard",jistkan:"jistka",jotun: "giant",jyoti:"celestial",infernal:"infernal",kelish:"kelish",mwangi:"azlanti",necril:"drowic",orc:"dwarvish",orcish:"dwarvish",polyglot:"azlanti",protean:"abyssal",requian:"requian",shoanti:"azlanti",skald:"jitska",sphinx:"requian",strix:"infernal",sylvan:"sylvan",shoony:"dwarvish",taldane:'azlanti',tengu:"oriental",terran:"dwarvish",thassilonian:"thassilonian",tien:"oriental",treant:"sylvan",undercommon:"drowic",utopian:"utopian",varisian:"azlanti",vegepygmy:"gnoll",vudrani:"garundi"}
+                break;
+            case "ose":
+                this.alphabets = {common:'130% Thorass',lawful:'180% Celestial',chaotic:'150% Barazhad',neutral:'230% Infernal',doppelganger:'250% Pulsian',dwarvish:'120% Dethek',draconic:'170% Iokharic',gargoyle:'150% HighDrowic',gnoll:'150% Kargi',gnomish:'120% Tengwar',harpy:'100% OldeThorass',elvish:'150% Espruar',ogre:'120% MeroiticDemotic',sylvan:'150% OldeEspruar'}
+                this.tongues = {_default:'common',1:'lawful',2:'chaotic',3:'neutral',4:'dwarvish',5:'doppelganger',6:'draconic',7:'dwarvish',8:'elvish',9:'gargoyle',10:'gnoll',11:'gnomish',12:'dwarvish',14:'harpy',15:'dwarvish',16:'draconic',17:'draconic',18:'gargoyle',19:'sylvan',20:'ogre',21:'dwarvish',22:'sylvan'}
+                break;
+            case "wfrp4e":
+            default:
+                this.alphabets = {common:'120% Dethek'}
+                this.tongues = {_default:'common'}
```

Comment:
These make for some really huge lines, I'm not sure if it would be better if they were made as a one line per entry : 
```js
this.alphabets = {
   common: '130% Thorass',
   celestial: '180% Celestial',
// etc...
}
```
With proper indentation and spaces between between the `:` character.
I think it might be huge though, so *maybe* move that definition to a separate file that gets imported as well ? 
What do you think ?

Reply:
I like it because that way even people who dont program javascript can simply read the files and give their feedback, make suggestions on alterations, on implementation of new systems, etc. An import/export feature can also be added later, lots of possibilities. We could save it into json format, Im thinking a different file to every 'language-system', similar to how some modules deal with language.

What do you think? Different files means loading the files wont be heavy, no matter how many systems we support.

Reply:
I just saw what you did with the json files that get loaded. That wasn't what I had in mind, but I love your idea, so yeay 👍